spec/lib/tasks/jobs.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+describe "jobs:populate_job_field" do
+  include_context "rake"
+
+  context "when the jobs are published" do
+    context "when there's no published_at date" do
+      let!(:jobs) { create_list(:job, 3, :published) }
+
+      before do
+        subject.invoke
+      end
+
+      it "populate the published_at date" do
+        expect(
+          jobs.all? { |job| job.published_at.present? }
```

Comment:
You can use a form like `expect(jobs.pluck(:published_at)).to all be_present`

The `all` helper is very useful for collections